src/components/ChangesetFileList/ChangesetFileList.js

```diff
@@ -68,27 +32,15 @@ const ChangesetFileList = ({ compact, files }: Props) =>
                       added={file.stats.added}
                       changed={0}  // FIXME
                     />
-
                   </Col>
                 }
                 <Col lg={8} md={8} sm={8} xs={8}>
-                  <div>
+                  <Link
+                    style={{ cursor: 'pointer' }}
+                    to={`${window.location.pathname}/${file.name}`}
```

Comment:
@kkateq do we have a URL for seeing a specific file from a PR?

Github links to the revision:
https://github.com/Unity-Technologies/Tanto/blob/dd811bc9c86ec94cdb69aafd328bbcb9c5650c21/src/components/ChangesetDelta/ChangesetDelta.css